Makefile

```diff
@@ -133,9 +132,12 @@ cluster/prepare/configmaps:
 cluster/prepare/osrc:
 	- oc process -p NAMESPACE=$(NAMESPACE) OPERATOR_SOURCE_REGISTRY_NAMESPACE=$(ORG) -f deploy/operator-source-template.yml | oc apply -f - -n openshift-marketplace
 
+.PHONY: cluster/prepare/crd
+cluster/prepare/crd:
```

Comment:
I'd consider changing this to `cluster/prepare/crds`, to correspond to `cluster/cleanup/crds`. WDYT?

Reply:
Yeah, thats good with me